version.txt

```diff
@@ -1,2 +1,2 @@
-0.86.0
+0.86.1
```

Comment:
Yeah... I found the release plugin to be buggy so I didn't use it last time. This should be ´0.86.1-SNAPSHOT`